src/Compiler/pars.fsy

```diff
@@ -3335,7 +3335,7 @@ atomicUnsignedRationalConstant:
             SynRationalConst.Integer(fst $1, lhs parseState) }
 
   | LPAREN rationalConstant rparen
-    { $2 }
+    { SynRationalConst.Paren($2, lhs parseState) }
```

Comment:
I can't fully explain why but here I would have gone with `rhs2 parseState 1 3` instead of `lhs`. I think that would work better when recovery is in play. I'm not a 100% sure though.

Reply:
Ah yeah, I remember something about that...
I need to document it.